{
  "affectedProducts": [
    "Rocky Linux 8"
  ],
  "cves": [
    {
      "cvss3BaseScore": "7.1",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:N/A:H",
      "cwe": "CWE-20->CWE-125",
      "name": "CVE-2020-35653",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-35653"
    },
    {
      "cvss3BaseScore": "5.4",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:N/A:L",
      "cwe": "CWE-122->CWE-125",
      "name": "CVE-2020-35655",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-35655"
    },
    {
      "cvss3BaseScore": "9.1",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H",
      "cwe": "CWE-125",
      "name": "CVE-2021-25287",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-25287"
    },
    {
      "cvss3BaseScore": "9.1",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H",
      "cwe": "CWE-125",
      "name": "CVE-2021-25288",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-25288"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-120",
      "name": "CVE-2021-25290",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-25290"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-20",
      "name": "CVE-2021-25292",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-25292"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-125",
      "name": "CVE-2021-25293",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-25293"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-20->CWE-400",
      "name": "CVE-2021-27921",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-27921"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-20->CWE-400",
      "name": "CVE-2021-27922",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-27922"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-20->CWE-400",
      "name": "CVE-2021-27923",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-27923"
    },
    {
      "cvss3BaseScore": "5.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
      "cwe": "CWE-20",
      "name": "CVE-2021-28675",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-28675"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-20->CWE-835",
      "name": "CVE-2021-28676",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-28676"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-20",
      "name": "CVE-2021-28677",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-28677"
    },
    {
      "cvss3BaseScore": "7.5",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-20",
      "name": "CVE-2021-28678",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-28678"
    },
    {
      "cvss3BaseScore": "5.9",
      "cvss3ScoringVector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
      "cwe": "CWE-119",
      "name": "CVE-2021-34552",
      "sourceBy": "MITRE",
      "sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-34552"
    }
  ],
  "description": "The python-pillow packages contain a Python image processing library that provides extensive file format support, an efficient internal representation, and powerful image-processing capabilities.\n\nSecurity Fix(es):\n\n* python-pillow: Out-of-bounds read in J2K image reader (CVE-2021-25287)\n\n* python-pillow: Out-of-bounds read in J2K image reader (CVE-2021-25288)\n\n* python-pillow: Negative-offset memcpy in TIFF image reader (CVE-2021-25290)\n\n* python-pillow: Regular expression DoS in PDF format parser (CVE-2021-25292)\n\n* python-pillow: Out-of-bounds read in SGI RLE image reader (CVE-2021-25293)\n\n* python-pillow: Excessive memory allocation in BLP image reader (CVE-2021-27921)\n\n* python-pillow: Excessive memory allocation in ICNS image reader (CVE-2021-27922)\n\n* python-pillow: Excessive memory allocation in ICO image reader (CVE-2021-27923)\n\n* python-pillow: Excessive memory allocation in PSD image reader (CVE-2021-28675)\n\n* python-pillow: Infinite loop in FLI image reader (CVE-2021-28676)\n\n* python-pillow: Excessive CPU use in EPS image reader (CVE-2021-28677)\n\n* python-pillow: Excessive looping in BLP image reader (CVE-2021-28678)\n\n* python-pillow: Buffer overflow in image convert function (CVE-2021-34552)\n\n* python-pillow: Buffer over-read in PCX image reader (CVE-2020-35653)\n\n* python-pillow: Buffer over-read in SGI RLE image reader (CVE-2020-35655)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nAdditional Changes:\n\nFor detailed information on changes in this release, see the Rocky Linux 8.5 Release Notes linked from the References section.",
  "fixes": [
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1915420",
      "ticket": "1915420"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1915432",
      "ticket": "1915432"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1934685",
      "ticket": "1934685"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1934699",
      "ticket": "1934699"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1934705",
      "ticket": "1934705"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1935384",
      "ticket": "1935384"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1935396",
      "ticket": "1935396"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1935401",
      "ticket": "1935401"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1958226",
      "ticket": "1958226"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1958231",
      "ticket": "1958231"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1958240",
      "ticket": "1958240"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1958252",
      "ticket": "1958252"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1958257",
      "ticket": "1958257"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1958263",
      "ticket": "1958263"
    },
    {
      "sourceBy": "Red Hat",
      "sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1982378",
      "ticket": "1982378"
    }
  ],
  "name": "RLSA-2021:4149",
  "publishedAt": "2021-11-09T08:24:34Z",
  "rpms": {
    "Rocky Linux 8": {
      "nvras": [
        "python3-pillow-0:5.1.1-16.el8.aarch64.rpm",
        "python3-pillow-0:5.1.1-16.el8.x86_64.rpm",
        "python3-pillow-debuginfo-0:5.1.1-16.el8.aarch64.rpm",
        "python3-pillow-debuginfo-0:5.1.1-16.el8.x86_64.rpm",
        "python3-pillow-tk-debuginfo-0:5.1.1-16.el8.aarch64.rpm",
        "python3-pillow-tk-debuginfo-0:5.1.1-16.el8.x86_64.rpm",
        "python-pillow-0:5.1.1-16.el8.src.rpm",
        "python-pillow-debuginfo-0:5.1.1-16.el8.aarch64.rpm",
        "python-pillow-debuginfo-0:5.1.1-16.el8.x86_64.rpm"
      ]
    }
  },
  "severity": "SEVERITY_MODERATE",
  "shortCode": "RL",
  "synopsis": "Moderate: python-pillow security update",
  "topic": "An update is available for python-pillow.\nThis update affects Rocky Linux 8.\nA Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE list",
  "type": "TYPE_SECURITY"
}
